packages/core/src/components/Table/index.ts

```diff
@@ -8,7 +8,6 @@ export { default as TableBody, ITableBodyProps as TableBodyProps } from "./Table
 export {
   default as TableVirtualizedBody,
   ITableVirtualizedBodyProps as TableVirtualizedBodyProps,
-  TableVirtualizedRows,
```

Comment:
Is it breaking?

Reply:
yes and no
we only just pushed that type recently, and I'm not sure we're exporting the types correctly right now, aren't we?